<ui version="4.0" >
 <class>TransformWidget</class>
 <widget class="QWidget" name="TransformWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Transform</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QToolBox" name="toolBox" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="currentIndex" >
      <number>2</number>
     </property>
     <widget class="QWidget" name="position" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>163</width>
        <height>137</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Position</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QGridLayout" >
         <item row="2" column="0" >
          <widget class="QLabel" name="m_posZLabel" >
           <property name="text" >
            <string>Z:</string>
           </property>
           <property name="buddy" >
            <cstring>m_posZ</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QDoubleSpinBox" name="m_posY" >
           <property name="decimals" >
            <number>3</number>
           </property>
           <property name="minimum" >
            <double>-100000.000000000000000</double>
           </property>
           <property name="maximum" >
            <double>100000.000000000000000</double>
           </property>
           <property name="singleStep" >
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="m_posXLabel" >
           <property name="text" >
            <string>X:</string>
           </property>
           <property name="buddy" >
            <cstring>m_posX</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="m_posYLabel" >
           <property name="text" >
            <string>Y:</string>
           </property>
           <property name="buddy" >
            <cstring>m_posY</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QDoubleSpinBox" name="m_posX" >
           <property name="decimals" >
            <number>3</number>
           </property>
           <property name="minimum" >
            <double>-100000.000000000000000</double>
           </property>
           <property name="maximum" >
            <double>100000.000000000000000</double>
           </property>
           <property name="singleStep" >
            <double>0.100000000000000</double>
           </property>
           <property name="value" >
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QDoubleSpinBox" name="m_posZ" >
           <property name="decimals" >
            <number>3</number>
           </property>
           <property name="minimum" >
            <double>-100000.000000000000000</double>
           </property>
           <property name="maximum" >
            <double>100000.000000000000000</double>
           </property>
           <property name="singleStep" >
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="orientation" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>137</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Orientation</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="m_rotXLabel" >
           <property name="text" >
            <string>X:</string>
           </property>
           <property name="buddy" >
            <cstring>m_rotX</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QDoubleSpinBox" name="m_rotX" >
           <property name="suffix" >
            <string/>
           </property>
           <property name="minimum" >
            <double>-359.990000000000009</double>
           </property>
           <property name="maximum" >
            <double>359.990000000000009</double>
           </property>
           <property name="singleStep" >
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="m_rotYLabel" >
           <property name="text" >
            <string>Y:</string>
           </property>
           <property name="buddy" >
            <cstring>m_rotY</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QDoubleSpinBox" name="m_rotY" >
           <property name="suffix" >
            <string/>
           </property>
           <property name="minimum" >
            <double>-359.990000000000009</double>
           </property>
           <property name="maximum" >
            <double>359.990000000000009</double>
           </property>
           <property name="singleStep" >
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="m_rotZLabel" >
           <property name="text" >
            <string>Z:</string>
           </property>
           <property name="buddy" >
            <cstring>m_rotZ</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QDoubleSpinBox" name="m_rotZ" >
           <property name="suffix" >
            <string/>
           </property>
           <property name="minimum" >
            <double>-359.990000000000009</double>
           </property>
           <property name="maximum" >
            <double>359.990000000000009</double>
           </property>
           <property name="singleStep" >
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="resolution" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>213</width>
        <height>584</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Resolution</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="pushButton_2" >
           <property name="maximumSize" >
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Remove current resolution level</string>
           </property>
           <property name="text" >
            <string>-</string>
           </property>
           <property name="flat" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton" >
           <property name="maximumSize" >
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Add resolution level</string>
           </property>
           <property name="text" >
            <string>+</string>
           </property>
           <property name="flat" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
